reference-architecture/azure-ansible/bastion.sh

```diff
@@ -178,9 +178,9 @@ EOF
 
 # Create Azure Cloud Provider configuration Playbook
 
-cat > /home/${AUSERNAME}/setup-azure-master.yml <<EOF
+cat > /home/${AUSERNAME}/azure-config.yml <<EOF
 #!/usr/bin/ansible-playbook
-- hosts: masters
+- hosts: all
   gather_facts: no
   serial: 1
```

Comment:
I think "serial" can be removed, because it can be done on all the hosts at the same time. Thoughts?